fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -451,6 +455,132 @@ void testMutualIndexingCrashAndAllowContinueNonMutually(boolean explicit) {
         assertAllValidated(indexes);
     }
 
+    @ParameterizedTest
```

Comment:
It seems like there should be a test exemplifying the behavior around SynchronizedSession, and how the mutual indexing doesn't respect it.
I believe if you use SynchronizedSession, and try to takeover from MULTI to SINGLE, while the former is still active due to the SynchronizedSession, but mutual will not do that, it will takeover even if the MULTI just committed a second ago. 

Perhaps this should be broken out into a separate body of work to improve the SynchronizedSession relationship with MUTUAL, but it might be pretty easy to add those tests, and then reference an issue to improve the situation.

Reply:
✅